### PR TITLE
feat: add compositors, power menu, dock and document viewer #92

### PR DESCRIPTION
**Pull request by nesper8**
sorry about making a new PR
i made so many mistakes on #92 so i have made a new branch upstream with aggregated list from patch-1 repo

### CODE REVIEW

**Review by nesper8 — COMMENTED** [review]
verified changes

**Comment by nesper8** [comment]
@mpsq can you please review and merge?

File: layouts/index.html [hunk 1]
```diff
@@ -101,6 +112,12 @@ <h2 class="title--answer">Mostly...</h2>
         <a href="https://gitlab.com/WhyNotHugo/shotman">shotman</a>,
         <a href="https://github.com/jtheoof/swappy">swappy</a>
       </li>
+      <li class="list__item--ok">
+        Stacking Compositor:
```
Comment (mpsq) [inline layouts/index.html:116]: Style nit:
```suggestion
        Stacking compositor:
```

File: layouts/index.html [hunk 2]
```diff
@@ -59,6 +66,10 @@ <h2 class="title--answer">Mostly...</h2>
         <a href="https://gitlab.gnome.org/GNOME/eog">Eye of GNOME</a>,
         <a href="https://github.com/eXeC64/imv">imv</a>
       </li>
+      <li class="list__item--ok">
+        Logout Menu:
```
Comment (mpsq) [inline layouts/index.html:70]: Can we rename to "Power menu"? I think it is better if a bit more generic

**Review by mpsq — CHANGES_REQUESTED** [review]
Tiny changes, looks good otherwise. Thanks for your contribution :)

File: layouts/index.html (hunk 2 above)
Comment (nesper8) [reply layouts/index.html:70]: yeah cool

**Comment by nesper8** [comment]
Coolio @mpsq

**Comment by nesper8** [comment]
Uh dude when you gonna commit these?

**Comment by mpsq** [comment]
I am just waiting for you to fix the 2 comments I added in the last review

**Comment by nesper8** [comment]
wait lmao i thought you changed it when you posted the recommendations

**Comment by nesper8** [comment]
btw it seems that you did change Stacking Compositor to Stacking compositor
idk what happened im new to git itself

**Review by mpsq — APPROVED** [review]
Awesome, thanks for your contribution :)

**Pull request merged**